### PR TITLE
k8sify

### PR DESCRIPTION
**Pull request by sam-qordoba**
* This works on minikube!
* `kubectl apply -f k8s/chime.yaml` spins up 15 pods
* if you proxy into the cluster (`kubectl proxy`),  you can access the app on http://127.0.0.1:8001/api/v1/namespaces/default/services/chime:80/proxy/

HOWEVER

* it uses the default namespace
* there are no resource requests/limits
* ingress is copy/pasted, I did not test - can someone tell me how to test ingress locally?

### CODE REVIEW

File: k8s/chime.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  selector:
+    matchLabels:
+      app: chime
+  template:
+    metadata:
+      labels:
+        app: chime
+    spec:
+      containers:
+      - image: qsam/chime:latest
```
Comment (sam-qordoba) [inline k8s/chime.yaml]:
```suggestion
      - image: codeforphilly/chime:latest
```

**Comment by lottspot** [comment]
> it uses the default namespace

App specific namespace added in 9def6ca

> there are no resource requests/limits

We're unconcerned with this since the app will be running on a dedicated cluster for the time being

> ingress is copy/pasted, I did not test - can someone tell me how to test ingress locally?

Tested this on the prospective cluster. Appears to be working ok.

**Comment by sam-qordoba** [comment]
@lottspot I am interested in understanding infra.yaml, if you have any resources. Obviously not blocking.

**Comment by lottspot** [comment]
@sam-qordoba tbh those are just copy-paste resources which @themightychris tracked down. In summary, there are two apps smashed into one file:

- [cert-manager](https://cert-manager.io/docs/) to automatically issue Let's Encrypt certs for Ingress resources
- [ingress-nginx](https://github.com/kubernetes/ingress-nginx) to deploy a backend implementation of the Ingress resource

**Comment by themightychris** [comment]
I think we should un-smash those at some point and keep each piece of infra separate so they can reference their upstream source and we can easily inspect in the git history if we editing anything before updating them from upstream

**Pull request merged**